_posts/2019-04-14-ferryman-post.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Our First Game Jam!"
+date:   2019-04-14 14:55:00 -0500
+categories: blog
+author: Aaron
+author_twitter: shamshirz
+image: /assets/ferryman-glass.png
+---
+_48 hour sprint to Unity greatness_
+[![Ferryman image]({{ "/assets/ferryman-glass.png" | relative_url }}){:class="excerpt-image"}]({{ site.baseurl }}{% post_url 2019-04-10-ferryman %})
+<!-- Ends the excerpt text, it includes the image -->
+
+
+How it works.
```

Comment:
"works" in the present tense makes it sound like this is how all game jams go, but "Spirits" is pretty unique to this one. maybe "worked"

Reply:
I thought of it as an attempt to pull me into that time, e.g. "You're sitting at long IKEA table in a studio apartment in Somerville, MA with your two bros, taking turns dipping a baguette into a container of chive cream cheese. Because that's just how it works."

Reply:
⚙️